packages/components/src/date-time/date/styles.ts

```diff
@@ -106,15 +125,14 @@ export const DayButton = styled( Button, {
 		props.hasEvents &&
 		`
 		::before {
-			background: ${ props.isSelected ? COLORS.white : COLORS.theme.accent };
+			border: 2px solid ${ props.isSelected ? COLORS.white : COLORS.theme.accent };
```

Comment:
Same nit re: theming. Similarly, the `COLORS.gray`s can be switched to `COLORS.theme.gray` if we still care. (Do we...? 😅)

```suggestion
			border: 2px solid ${ props.isSelected ? COLORS.theme.accentInverted : COLORS.theme.accent };
```

Reply:
Why not 😅